### PR TITLE
add cz.ondrejkolin.Barcoder

### PR DESCRIPTION
**Pull request by ondrejkolin**
### Please confirm your submission meets all the criteria

It's a simple barcode generator. 

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: cz.ondrejkolin.Barcoder.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+id: cz.ondrejkolin.Barcoder
+runtime: org.gnome.Platform
+runtime-version: "45"
+sdk: org.gnome.Sdk
+command: barcoder
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
```
Comment (hfiguiere) [inline cz.ondrejkolin.Barcoder.yaml:10]: --share=ipc is missing

File: cz.ondrejkolin.Barcoder.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+id: cz.ondrejkolin.Barcoder
+runtime: org.gnome.Platform
+runtime-version: "45"
```
Comment (hfiguiere) [inline cz.ondrejkolin.Barcoder.yaml:4]: 46 is out

File: cz.ondrejkolin.Barcoder.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+id: cz.ondrejkolin.Barcoder
```
Comment (hfiguiere) [inline cz.ondrejkolin.Barcoder.yaml:2]: `http://ondrejkolin.cz/` return a 403

**Comment by hfiguiere** [comment]
bot, build cz.ondrejkolin.Barcoder

File: cz.ondrejkolin.Barcoder.yaml (hunk 3 above)
Comment (ondrejkolin) [reply cz.ondrejkolin.Barcoder.yaml:2]: Nothing there still, but at least a valid 200 and working gitlab pages.

**Comment by ondrejkolin** [comment]
The bot failed, I will fix it tomorrow... Thanks for the input!

File: cz.ondrejkolin.Barcoder.yaml (hunk 2 above)
Comment (ondrejkolin) [reply cz.ondrejkolin.Barcoder.yaml]: fixed

**Comment by ondrejkolin** [comment]
bot, build cz.ondrejkolin.Barcoder

**Comment by bbhtt** [comment]
https://gitlab.com/ondrej.kolin/barcoder/-/blob/1738d5705f44e83c7581317804709ee39dbff5b7/data/cz.ondrejkolin.Barcoder.metainfo.xml.in#L5 should be reverse DNS

**Comment by ondrejkolin** [comment]
bot, build cz.ondrejkolin.Barcoder

@bbhtt I've fixed that now

**Pull request closed without merging**